comp/otelcol/otlp/components/exporter/logsagentexporter/logs_exporter.go

```diff
@@ -73,18 +73,22 @@ func (e *Exporter) ConsumeLogs(ctx context.Context, ld plog.Logs) (err error) {
 		if ddLog.Service != nil {
 			service = *ddLog.Service
 		}
-		status := ddLog.AdditionalProperties["status"]
-		if status == "" {
-			status = message.StatusInfo
+		status := message.StatusInfo
+		if val, ok := ddLog.AdditionalProperties["status"]; ok {
+			if strVal, ok := val.(string); ok && strVal != "" {
```

Comment:
this is because `AdditionalProperties` changes from `map[string]string` to `map[string]interface{}` in the upstream dependency